src/App.js

```diff
@@ -1,25 +1,107 @@
-import logo from './logo.svg';
 import './App.css';
+import {useMemo,useState } from "react";
 
-function App() {
+function Square(props) {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+      <button className="square" onClick={props.onClick}>
+        {props.value}
+      </button>
+    );
 }
 
-export default App;
+function Board(props) {
+  
+    // const rendersquare = props.squares.map((item ,index) =>
```

Comment:
if it is not necessary feel free to delete any part of the code. (you can left some for future reference though)